libs/model/src/community/GetCommunity.query.ts

```diff
@@ -0,0 +1,29 @@
+import { type Query } from '@hicommonwealth/core';
+import * as schemas from '@hicommonwealth/schemas';
+import { Includeable } from 'sequelize';
+import { models } from '../database';
+
+export function GetCommunity(): Query<typeof schemas.GetCommunity> {
+  return {
+    ...schemas.GetCommunity,
+    auth: [],
+    secure: false,
+    body: async ({ payload }) => {
+      const where = { id: payload.id };
+      const include: Includeable[] = [];
+      if (payload.include_node_info) {
+        include.push({
+          model: models.ChainNode,
+          required: true,
+        });
+      }
+
+      return (
+        await models.Community.findOne({
+          where,
+          include,
+        })
+      )?.toJSON();
```

Comment:
Do we want to throw an error if the community isn't found?

Reply:
We should return a null value if not found and let the client handle the exception, unless you see a reason to throw an error in the route itself.

Reply:
I haven't seen us use this convention before, but then again we rarely ever return a single entity. Seems fine to me though– less spam in the logs.